easybuild/easyconfigs/g/GAMESS-US/GAMESS-US-20170420-R1_recent-gcc.patch

```diff
@@ -0,0 +1,52 @@
+--- config.orig	2017-07-18 19:34:32.056678893 +0000
```

Comment:
Please include a short description above this line + name of author or reference to upstream patch.

Maybe also include other GCC version number like 6.1, 6.2, 6.3, 6.4, 7.1 & 7.2 so this patch can be reused with other toolchains?